src/my_crypto.erl

```diff
@@ -4,7 +4,7 @@
 
 -export([hash/1, sign/2, verify/3, read_file_key/2, potato_key/0, is_public_key/1]).
```

Comment:
this file really needs to be renamed to potato_crypto

Reply:
at this point..